src/UI/examples/Chart/Speedo/Standard/maximum_configuration.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Max Example for rendering a standard Speedo with maximum configuration
+ */
+function maximum_configuration() {
+    //Loading factories
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    //Genarating and rendering the standard speedo
+    $speedo = $f->chart()->speedo()->standard(array(
+        'goal' => 400,
+        'score' => 250,
+        'minimum' => 300,
+        'diagnostic' => 200,

```

Comment:
Maybe use values that are a little easier to read in this example (since it will be the one most of the readers will consult first). Such as:
```
    $speedo = $f->chart()->speedo()->standard(array(
        'goal' => 100,
        'score' => 80,
        'minimum' => 75,
        'diagnostic' => 60,
    ));
```